.github/workflows/test.yaml

```diff
@@ -39,3 +39,22 @@ jobs:
           name: test-results
           path: ./test-results
           if-no-files-found: ignore
+
+  test-unit:
+    name: unite test
```

Comment:
`unite` -> `unit`  👀

Also, can we run `pnpm test:store` in the root directory, instead of `pnpm test:unit` in `packages/store`? This can include custom store E2E tests instead of unit tests.

Reply:
I thought you don't wanna run the e2e test under store